### ISSUE TITLE
Object detection metrics

### ISSUE THREAD

**Issue by WingRS** [open-edge-platform/geti#589]
Hi all, the mobilenet ssd example is working nicely for training, but there are two metrics that would be nice to have. 
1. Accuracy of the class prediction
2. IoU of the bounding boxes.
3. 
These two metrics are much of a help when trying to understand if the model is not going into overfitting even when the loss is going down on training.

It would be nice to see them both in train and in validation in tensorboard. I happen to have experience with developing such, but the code is so much incapsulated that it is hard to understand where to start.

**Comment by Ilya-Krylov** [comment]
Actual implementation of training/validation is inside of our mmdetection fork. You can have a look at this method and probably it will help you to add more metrics https://github.com/openvinotoolkit/mmdetection/blob/ote/mmdet/datasets/coco.py#L390

**Comment by WingRS** [comment]
Nice, thanks for the fast answer! Could you please also point me to the training script that is running the process itself?

**Comment by Ilya-Krylov** [comment]
https://github.com/openvinotoolkit/training_extensions/blob/develop/ote/ote/modules/trainers/base.py

**Comment by WingRS** [comment]
I am closing this issues, and will add a PR once I'll finish the update. Thanks for the answers!
